documentation/ame-guide/modules/ROOT/pages/generate/generate-openapi-doc.adoc

```diff
@@ -2,22 +2,23 @@
 
 To generate an OpenAPI specification from the Aspect Model, proceed as follows:
 
-* Click the *Generate* on xref:getting-started/ui-overview.adoc#menu-generate[menu]. +
-A menu offers one option to generate the OpenAPI specification. +:
-
-** *OpenAPI specification* -- continue with this one
+* Click the xref:getting-started/ui-overview.adoc#menu-generate[Generate menu] and select *OpenAPI Specification*.
 +
-image::generate-api-spec.png[Generate document, width=100%, align="left"]
+image:generate-api-spec.png[Generate document, width=15%]
+
 * In case your model supports multiple languages, select the specific language for the documentation.
 * Select *JSON* or *YAML* as the desired file format.
 * Set the *Base URL* were the Aspect API will be served from.
```

Comment:
were = where?
Base URL is written BaseUrl in the UI; I am not sure if we also format entry fields in bold? (if this is consistent in the documentation, I am fine 😊)

Reply:
As we write the checkboxes as in the UI, I would do the same for BaseUrl.